services/QuillLMS/app/controllers/api/v1/concept_feedback_controller.rb

```diff
@@ -4,12 +4,9 @@ class Api::V1::ConceptFeedbackController < Api::ApiController
   before_action :activity_type, except: [:index]
   before_action :concept_feedback_by_uid, except: [:index, :create, :update]
 
-  caches_action :index, format: 'json', expires_in: 1.hour
   def index
-    all_concept_feedbacks = ConceptFeedback
-      .where(activity_type: params[:activity_type])
-      .all
-      .reduce({}) { |agg, q| agg.update({q.uid => q.as_json}) }
+    all_concept_feedbacks = $redis.get(ConceptFeedback::ALL_CONCEPT_FEEDBACKS_KEY)
```

Comment:
I've decided to change my approach and get rid of caching on both of these endpoints instead of using redis (see Peter's comment below). So we won't need to use these keys anymore!